packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.scss

```diff
@@ -110,6 +110,9 @@ $DropDown-item-height: 36px;
       border-color: $ms-color-contrastWhiteDisabled;
       color: $ms-color-contrastWhiteDisabled;
     }
+    &.error {
```

Comment:
State/modifier classes like this should be in the format `elementIsState`. So in this case `titleIsError`